src/metabase/upload.clj

```diff
@@ -274,11 +259,11 @@
   (->> rows
        (map #(row->value-types % settings))
        (reduce coalesce-types (repeat column-count nil))
-       (map (fn [type]
+       (map (fn [tag]
```

Comment:
Using terminology from the hierarchy abstraction here avoids shadowing `clojure.core/type`